### PR TITLE
005/create pages with router

### PR DESCRIPTION
**Pull request by trandromeda**

### CODE REVIEW

**Comment by trandromeda** [comment]
@jondlove Code review please

File: src/Scroll.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, withRouter } from 'react-router-dom'
+
+class ScrollToTop extends React.Component {
+  componentDidUpdate(prevProps) {
+    const nav = document.getElementById("email-bar");
+    console.log(this.props.location)
```
Comment (jondlove) [inline src/Scroll.js]: Don't include console.logs in code that will become prod

File: src/App.js [hunk 2]
```diff
@@ -24,30 +25,19 @@ class App extends Component {
...
-            <LessonCard />
-            <LessonCard />          
-          </div>
-
-          <div className="preview-description">
-            <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et ligula ligula. Nullam pharetra sapien eget magna semper accumsan. Pellentesque lobortis tincidunt gravida. Integer iaculis velit in augue pellentesque, vitae cursus nunc laoreet. Vivamus justo nulla, efficitur sed nulla id, tincidunt vehicula lectus. Nunc tempus rhoncus porttitor. Fusce vitae venenatis nunc. Sed ornare scelerisque neque ut euismod. Cras id scelerisque ipsum, eleifend euismod libero. Maecenas dignissim ut diam quis ornare. Duis vehicula vehicula imperdiet.
-            </p>
-          </div>
+        <div className="navbar" id="navbar">
```
Comment (jondlove) [inline src/App.js]: Should be a component

File: src/components/About.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+        <p>
+        In doing so, we will also strive to:
+        </p>
+
+        <ul>
+          <li>Preserve cultural knowledge that may otherwise be lost to time,</li>
+          <li>Encourage people to do social good,</li>
+          <li>Provide a form of financial gain to seniors, and</li>
+        <li>Tap into the underutilized resources possessed by an entire generation</li>
```
Comment (jondlove) [inline src/components/About.js]: Minor whitespace alignment issue

File: src/components/About.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+
+class About extends Component {
+  render() {
+    return (
+        <div className="about" id="main">
```
Comment (jondlove) [inline src/components/About.js]: Your div here has a different alignment than used in ComingSoon (i.e. in other components all the child elements are indented)

File: src/components/ComingSoon.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+class ComingSoon extends Component {
+  render() {
+    return (
+        <div className="coming-soon">
```
Comment (jondlove) [inline src/components/ComingSoon.js:6]: Missing `id="main"` if consistent with other components (or remove from other components)

File: src/components/Home.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+import EmailBar from './EmailBar';
+
+class Home extends Component {
+  render() {
+    return (
+        <div className="home" id="main">
+          <div className="preview">
+            <h1>Preview</h1>
+            <div className="lesson-container">
+              <LessonCard />
```
Comment (jondlove) [inline src/components/Home.js]: Are these `<LessonCard />` always fixed, or should they be populated from a list? [even for placeholder purposes]

File: src/Scroll.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, withRouter } from 'react-router-dom'
+
+class ScrollToTop extends React.Component {
```
Comment (jondlove) [inline src/Scroll.js]: Ideally filename matches classname; not strictly necessary, but adds consistency.

File: src/styles/App.scss [hunk 8]
```diff
@@ -99,14 +108,16 @@ a {
   }
 }
 
+$landing-height: 90vh;
```
Comment (jondlove) [inline src/styles/App.scss]: This should probably be at the top. If it's specific to this component, then this component's styling should be broken into a new file

**Review by jondlove — CHANGES_REQUESTED** [review]

**Review by jondlove — APPROVED** [review]

**Pull request merged**